app/build.gradle

```diff
@@ -35,7 +35,10 @@ android {
 
   productFlavors {
 
-    playground {} // Reserved for development
+    playground {
+      // Reserved for development
+      ext.enableCrashlytics = false
+    }
```

Comment:
what's the idea behind that?
Crashlytics should be an additional  / optional dependency every build flavor ...